### DISCUSSION TITLE
Future use of Context API in Web Client

### DISCUSSION THREAD

**Discussion by Tom-stack3** [TomBenDor/AP2#2] [Ideas]
I think it will be helpful to use the React Context API in order to pass information between certain components in our Web Client.
So it's worth reading about it a bit.
[Official documentation](https://reactjs.org/docs/context.html).

**Comment by TomBenDor** [comment]
That shall indeed be useful. Thanks Tommy 😸 

Regards,
Tom

**Comment by Tom-stack3** [comment]
It wasn't so useful after all :)
